src/main/default/classes/HandlerConfig.cls

```diff
@@ -0,0 +1,75 @@
+/**
+ * Class used to store all things related to a handler setup
```

Comment:
Moved this to it's own class and incorporated the ` global Handler handler;` indirectly into it (gets set after by the manager).

This makes it much easier to pass things around